CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,6 @@
-- [cosmos] [HARDENING] Add documentation about how to consume context data from Sinfonier (#145)
-- [cosmos] [HARDENING] Add Apiary documentation (#177)
-- [cosmos] [HARDENING] Add improvements to data_copier.sh (#180)
+- [cosmos][hardening] Add documentation about how to consume context data from Sinfonier (#145)
+- [cosmos][hardening] Add Apiary documentation (#177)
+- [cosmos-tidoop-api][feature] Add suport for MR job input parameters (#181)
+- [cosmos][hardening] Add documentation about how to consume context data from Sinfonier (#145)
+- [cosmos-tidoop-api][hardening]] Add Apiary documentation (#177)
+- [cosmos-admin][hardening] Add improvements to data_copier.sh (#180)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/183/commits/2cd16c45b70422b77ed2cec1720a89e23630f3c2